src/main.c

```diff
@@ -610,6 +597,40 @@ int main(int argc, char *argv[])
     }
   }
 
+  //
+  //  OK, now we can process the sysout_name
+  //  Order of priority:
+  //    1. Value of -sysout command line arg
+  //    2. Value of the first command line arg
```

Comment:
Though because of the way the X11 options parser works, it's actually the first non-X11 accepted argument.  This will work perfectly:
```
lde -title "foo" lisp.sysout -geometry 500x500+20+20
```